### PR TITLE
[16.0][ADD] pos_daily_sales_reports_category_only

### PR DESCRIPTION
**Pull request by trisdoan**

### CODE REVIEW

**Comment by santostelmo** [comment]
The code LG,, but any chance to add some test coverage ?

File: pos_daily_sales_reports_category_only/models/pos_daily_sales_reports.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                .mapped("sales_report_by_category_only")
+            )
+        else:
+            sales_report_by_category_only = any(
+                self.env["pos.session"]
+                .search([("id", "in", session_ids)])
+                .mapped("config_id.sales_report_by_category_only")
+            )
+        report_vals["sales_report_by_category_only"] = sales_report_by_category_only
+        return report_vals
```
Comment (santostelmo) [inline pos_daily_sales_reports_category_only/models/pos_daily_sales_reports.py:33]: You could add a test to ensure that`report_vals` has `sales_report_by_category_only` set

Comment (trisdoan) [reply pos_daily_sales_reports_category_only/models/pos_daily_sales_reports.py:33]: Hello @santostelmo, I added the test :pray:

**Review by cyrilmanuel — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]

**Review by cheoreyes — COMMENTED** [review]
Funcional ok

**Review by ivantodorovich — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**